include/types.h

```diff
@@ -44,7 +44,7 @@ struct Controller {
   /*0x18*/ OSContStatus *statusData;
   /*0x1C*/ OSContPad *controllerData;
 #if ENABLE_RUMBLE
-  /*0x1C*/ s32 port;
+  /*0x1E*/ s32 port;
```

Comment:
`0x20` :)

Reply:
if that's supposed to be 0x20, then the value was wrong to begin with, as `controllerData` also went up by only 2

Reply:
wait no it didnt lol